commit 37f373e33496ea437cc7e375cc835c20d4b35fb2
Author: H.J. Lu <hjl.tools@gmail.com>
Date:   Thu Feb 10 11:52:50 2022 -0800

    x86-64: Remove bzero weak alias in SS2 memset
    
    commit 3d9f171bfb5325bd5f427e9fc386453358c6e840
    Author: H.J. Lu <hjl.tools@gmail.com>
    Date:   Mon Feb 7 05:55:15 2022 -0800
    
        x86-64: Optimize bzero
    
    added the optimized bzero.  Remove bzero weak alias in SS2 memset to
    avoid undefined __bzero in memset-sse2-unaligned-erms.
    
    (cherry picked from commit 0fb8800029d230b3711bf722b2a47db92d0e273f)

diff --git a/sysdeps/x86_64/multiarch/memset-sse2-unaligned-erms.S b/sysdeps/x86_64/multiarch/memset-sse2-unaligned-erms.S
index bac74ac37fd3c144..2951f7f5f70e274a 100644
--- a/sysdeps/x86_64/multiarch/memset-sse2-unaligned-erms.S
+++ b/sysdeps/x86_64/multiarch/memset-sse2-unaligned-erms.S
@@ -31,9 +31,7 @@
 # endif
 
 # undef weak_alias
-# define weak_alias(original, alias) \
-	.weak bzero; bzero = __bzero
-
+# define weak_alias(original, alias)
 # undef strong_alias
 # define strong_alias(ignored1, ignored2)
 #endif
